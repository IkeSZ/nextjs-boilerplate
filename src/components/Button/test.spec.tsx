import { screen, render } from '@testing-library/react';

import Button from './index';

describe('<Button />', () => {
  it('should be able to render the heading', () => {
    const { container } = render(<Button />);

    expect(
      screen.getByRole('heading', {
        name: 'Button',
      })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
