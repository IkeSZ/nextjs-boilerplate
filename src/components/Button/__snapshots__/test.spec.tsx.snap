// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should be able to render the heading 1`] = `
<div
  class="styles__Container-mrxr17-0"
>
  <h1>
    Button
  </h1>
</div>
`;
